src/components/ApplicationViews.js

```diff
@@ -1,46 +1,26 @@
-import React, { Component } from "react";
-import { Route, Redirect } from "react-router-dom"
-import Login from "./auth/Login";
-import CreateAccount from "./auth/CreateAccount";
-import Home from "./songs/SongCollection";
-import SongView from "./songs/Song";
+import React, { Fragment } from 'react'
+import { Route, Redirect } from 'react-router-dom'
+import Login from './auth/Login'
+import { CreateAccount } from './auth/CreateAccount'
+import Home from './songs/SongCollection'
+import SongView from './songs/Song'
 
-class ApplicationViews extends Component {
+const ApplicationViews = ({ isAuthenticated, setUser, ...props }) => (
+  //If a user is logged in then render Home, otherwise render Login
+  <Fragment>
+    <Route
+      exact
+      path="/"
+      render={props => (isAuthenticated() ? <Home {...props} /> : <Redirect to="/login" />)}
```

Comment:
Arrow functions allow for implicit returns... no need to have big ole curlies with `return` statements everywhere.  I can talk all day about the myriad benefits of implicit returns, their limitations, and how their limitations force you to write better code.  So ask me about it 😄

Reply:
Also using ternary _expressions_ instead of if _statements_... ternaries _always_ return one thing... this is a tenant of "functional programming", and goes with the same benefits I alluded to about arrow functions.